internal/gh/rest.go

```diff
@@ -0,0 +1,164 @@
+package gh
+
+import (
+	"net/http"
+    "regexp"
+    "log/slog"
+    "encoding/json"
+)
+
+const baseURL = "https://api.github.com"
+var NumberForcePush = 0
```

Comment:
global counters like these have multiple drawbacks.
Most notability you introduce an implicit lifecycle with this approach. The output of each method using this variable changes when it is called multiple times. This is very counterintuitive as you can't observe / understand this behavior from outside the method.
you should use the counter as a local variable in the method called from outside this file. Then this variable is automatically reset every time the method is called